app/graphql/queries/authorized_model_query.rb

```diff
@@ -0,0 +1,44 @@
+module Queries
+  class AuthorizedModelQuery
+    def initialize(model_class:, user:)
+      @model_class = model_class
+      @user = user
+    end
+
+    delegate :find, :find_by, to: :authorized_scope
```

Comment:
I do not see any usage of `find` method. I think we should leave only `find_by` as we use that in `FetchField`.